### PR TITLE
Correct minor language issues

### PR DESCRIPTION
**Pull request by DinikaSen**
## Purpose
Correct language issues in authentication-session-persistence.md

### CODE REVIEW

File: en/docs/learn/authentication-session-persistence.md [hunk 1]
```diff
@@ -64,51 +64,51 @@ into either the service provider or the WSO2 Identity Server.
...
 instead of that we add a new record to the same session id with status
 called logout. So the valid record is the last one and all the other
 records under that session id will be outdated. Those outdated records
-are belong to the Operational Data.
+belong to the Operational Data.
 
 #### 3. Temporary Data
 
-In authentication flow, there are many temporary data object that will
+In authentication flow, there are many temporary data objects that will
```
Comment (gomathyk) [inline en/docs/learn/authentication-session-persistence.md]: there are many temporary data objects that are kept for a few seconds only.

File: en/docs/learn/authentication-session-persistence.md [hunk 2]
```diff
@@ -64,51 +64,51 @@ into either the service provider or the WSO2 Identity Server.
...
 
 There are three type of data objects that are persisted in the database.
 
 #### 1. Session Data
 
-Once the user get authenticated over WSO2 Identity Server, it will
-create a Session data object which is stored the authenticated user and
-the other authentication flow details. This will store in the database
-to share across the cluster nodes.
+Once the user gets authenticated over WSO2 Identity Server, it will
```
Comment (gomathyk) [inline en/docs/learn/authentication-session-persistence.md]: Once the user is authenticated, WSO2 Identity Server creates a session data object that stores ..

File: en/docs/learn/authentication-session-persistence.md [hunk 3]
```diff
@@ -64,51 +64,51 @@ into either the service provider or the WSO2 Identity Server.
...
+create a session data object which stores the authenticated user and
+the other authentication flow details. This will be stored in the database
+to be shared across the cluster nodes.
 
 #### 2. Operational Data
 
-This is covered the same session data as in above but the outdated
-records. Ex: Once user get authenticated, there will be a record for
-login status to that session id. Then again that user get logout from
+This covers the same session data as in above but the outdated
```
Comment (gomathyk) [inline en/docs/learn/authentication-session-persistence.md]: Shall we remove the first line and start from 

Once the user is authenticated, there will be a record for the login status for that session id. However, when that user is logged out from the system, we are not removing the above record from the table and
instead, we add a new record to the same session id with a status called logout. So the valid record is the last one and all the other records under that session id will be outdated. Those outdated records belong to the Operational Data.

File: en/docs/learn/authentication-session-persistence.md (hunk 1 above)
Comment (DinikaSen) [reply en/docs/learn/authentication-session-persistence.md]: Changed

File: en/docs/learn/authentication-session-persistence.md (hunk 2 above)
Comment (DinikaSen) [reply en/docs/learn/authentication-session-persistence.md]: Changed

File: en/docs/learn/authentication-session-persistence.md (hunk 3 above)
Comment (DinikaSen) [reply en/docs/learn/authentication-session-persistence.md]: Removed the first line

**Pull request merged**